src/tools/clippy/tests/ui/iter_count.fixed

```diff
@@ -3,11 +3,11 @@
 
 #![warn(clippy::iter_count)]
 #![allow(
-    unused_variables,
-    array_into_iter,
-    unused_mut,
-    clippy::into_iter_on_ref,
-    clippy::unnecessary_operation
+unused_variables,
```

Comment:
This change and the same change in the `rs` file will fail Clippy tests. Please re-indent those.

Reply:
Fixed.